packages/shorebird_cli/lib/src/validators/storage_access_validator.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:mason_logger/mason_logger.dart';
+import 'package:shorebird_cli/src/platform.dart';
+import 'package:shorebird_cli/src/process.dart';
+import 'package:shorebird_cli/src/validators/validators.dart';
+
+class StorageAccessValidator extends Validator {
+  @override
+  String get description => 'Has access to storage.googleapis.com';
+
+  @override
+  Future<List<ValidationIssue>> validate() async {
+    final result = await process.run(
```

Comment:
Shouldn't we make an `http` request using `package:http` since that's what we'll be using when users try to upload/download files instead of `ping`?